src/tsx/coreConnection.tsx

```diff
@@ -82,6 +105,7 @@ class DummyCoreConnection extends EventTarget implements ICoreConnection {
     this.dispatchEvent(
       new CustomEvent(eventNames.script, { detail: this.script }),
     );
+    this.sendEffectsChangedEvent();
```

Comment:
The name 'sendEffectsChangedEvent' is not very accurate in this context. Should it be just 'sendEffectsEvent'?